start/scaling-up/preparing-to-scale-up.md

```diff
@@ -113,15 +105,43 @@ memory (i.e. RAM) from that computer as a starting point. For instance,
 most laptop computers these days will have 8 or 16 GB of memory, which is okay to start 
 with if you know a single job will succeed on your laptop.
 
-	- For your initial tests it is OK to request more memory than 
+    - For your initial tests it is OK to request more memory than 
 your job may need so that the test completes successfully. **The key 
 is to adjust memory requests for subsequent jobs based on the results 
 of these test jobs.**
 
-	- If you find that memory usage will vary greatly across a 
+    - If you find that memory usage will vary greatly across a 
 batch of jobs, we can assist you with creating dynamic memory requests 
 in your submit files.
 
+# Optimize Job Resource Requests For Subsequent Jobs
+
+**As always, reviewing the HTCondor `log` file from past jobs is 
+a great way to learn about the resource needs of your jobs.** Optimizing the resources requested for each job may help your job run faster and achieve more throughput. 
+
+Save the HTCondor `log` files from your jobs. HTCondor will report 
+the memory, disk, and cpu usage of your jobs at the end of this file. The amount of each resource requested in the submit file is listed under the "Request" column and information about the amount of each resource actually utilized to complete the job is provided in the "Usage" column. 
+
+For example:
+
+```
+        Partitionable Resources :    Usage  Request Allocated
+           Cpus                 :                 1         1
+           Disk (KB)            :       12  1000000  26703078
+           Memory (MB)          :        0     1000      1000
+```
+
+
+-  One quick option to query your `log` files is to use the Unix tool `grep`. For example:
+    ```
+    [user@login]$ grep "Disk (KB)" my-job.log
+    ```
+    The above will return all lines in `my-job.log` that report the disk 
+    usage, request, and allocation of all jobs reported in that log file.
+
+    Alternatively, `condor_history` can be used to query details from 

```

Comment:
Could we add some more information about `condor_history` here? we don't have any of our own guides (🤦‍♀️ ), so either the manual or may just a note that some of hte options for `condor_q` below also work with `condor_history`?

Reply:
I added "Also works with condor_history" to the table for options that work with condor_history. If we want more detail for using condor_history, I can add it to my troubleshooting guide and then if we think it works better here, we can just move it back here.